src/localization.js

```diff
@@ -201,7 +201,7 @@ angular.module('ngLocalize')
         }
 
         function applySubstitutions(text, subs) {
-            var res = text,
+            var res = (angular.isArray(text) ? text.join('') : text),
```

Comment:
It's weird to have trailing whitespace in strings you wish to join, is there a specific reason why you can't remove the whitespace from your string array in the json and let the joiner do the work? I think that's the 99% use case anyway.